src/index.js

```diff
@@ -2,8 +2,11 @@ import angular from 'angular';
 import {AppModule} from 'metabolica';
 import {LoginModule} from './login/login.module';
 export {LoginModule} from './login/login.module';
+import {SessionModule} from './session/session.module';
+export {SessionModule} from './session/session.module';
```

Comment:
are you sure you need this syntax? exporting it normally once it's imported doesn't work properly?

Reply:
No I'm not sure, what do you mean by exporting it normally? It is exported for use [in decaf-metabolica](https://github.com/DD-DeCaF/decaf-metabolica/pull/25/files#diff-1fdf421c05c1140f6d71444ea2b27638R22).

Reply:
I guess you could just export it like:

    export SessionModule

Since it's already imported.

Reply:
Ah ok. Yes that would work. I don't have a strong opinion on this, I am just following the pattern from our other services.